checks.nix

```diff
@@ -37,6 +37,7 @@ in
   init-example-el = self.outputs.package.${system} {
     doomPrivateDir = ./test/doom.d;
     dependencyOverrides = inputs;
+    emacsPackages = with pkgs; emacsPackagesFor emacsNativeComp;
```

Comment:
Using non NativeComp Emacs is broken right now, but I think this is an upstream issue.

Reply:
Opened an issue upstream: https://github.com/doomemacs/doomemacs/issues?q=is%3Aissue+native-comp-eln-load-path+is%3Aclosed

Reply:
Perhaps we should wait for upstream to fix it then. No point in shipping a broken version.